### PR TITLE
Prepare documentation and such for v0.8 release

### PR DESCRIPTION
**Pull request by atomb**
Currently, this includes:
* Add `CHANGES.md` entries for v0.8
* Update some version notes in `README.md`

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -15,17 +15,125 @@ When the path to Java is known, SAW can automatically add system-related
...
+  `llvm_struct` is now a synonym for `llvm_alias`. The `llvm_struct`
+  function continues to be available for now.
+
+* The `llvm_pointer : LLVMType -> LLVMType` function allows construction
+  of arbitrary LLVM pointer types.
+
+* Two new functions, `llvm_points_to_at_type` and
+  `llvm_conditional_points_to_at_type`, mirror `llvm_points_to` and
+  `llvm_conditional_points_to`, but cast the pointer to a different
+  type. This may be useful when reading or writing a prefix of larger
```
Comment (RyanGlScott) [inline CHANGES.md:58]:
```suggestion
  type. This may be useful when reading or writing a prefix of a larger
```

File: CHANGES.md [hunk 2]
```diff
@@ -15,17 +15,125 @@ When the path to Java is known, SAW can automatically add system-related
...
+features:
+
+* Performance of JVM verification is significantly better, as a result
+  of removing some unnecessary instances of rewriting. This improves
+  performance of LLVM verification, as well.
+
+* The new `jvm_static_field_is` function allows describing the contents
+  of static variables in method specifications.
+
+* The verification code initializes all JVM classes at start so that
```
Comment (RyanGlScott) [inline CHANGES.md:29]:
```suggestion
* The verification code initializes all JVM classes at the start so that
```

File: CHANGES.md [hunk 3]
```diff
@@ -15,17 +15,125 @@ When the path to Java is known, SAW can automatically add system-related
...
+  initializers don't run at arbitray intermediate points and and clobber
```
Comment (RyanGlScott) [inline CHANGES.md:30]:
```suggestion
  initializers don't run at arbitrary intermediate points and clobber
```

File: CHANGES.md [hunk 4]
```diff
@@ -15,17 +15,125 @@ When the path to Java is known, SAW can automatically add system-related
...
+* Support for assuming LLVM contracts directly (rather than the previous
+  behavior which would temporarily assume that failed verifications
+  succeeded to determine whether higher-level verifications would still
+  succeed).
+
+* Support for global variables and initializers.
+
+* Support for null pointers.
+
+* Support for array value literals.
```
Comment (RyanGlScott) [inline CHANGES.md:95]: Some other PRs from the now-defunct `galois-py-toolkit` repo that might be worth mentioning:

* Support for specifying alignment in `alloc()`: https://github.com/GaloisInc/galois-py-toolkit/pull/17
* Support for accessing pointers by element index or field name: https://github.com/GaloisInc/galois-py-toolkit/pull/9

Comment (atomb) [reply CHANGES.md:95]: Ah, thanks! I'll include those.

**Pull request merged**